scripts/release/build/image_build_process.py

```diff
@@ -109,6 +115,28 @@ def check_if_image_exists(self, image_tag: str) -> bool:
         else:
             return True
 
+    def get_manfiest_list_digest(self, image) -> Optional[str]:
+        SKOPEO_IMAGE = "quay.io/skopeo/stable"
+
+        skopeo_inspect_command = ["inspect",  f"docker://{image}", "--format", "{{.Digest}}"]
+        docker_run_skopeo = ["docker", "run", "--rm", SKOPEO_IMAGE]
+        docker_run_skopeo.extend(skopeo_inspect_command)
+        
+        try:
+            result = subprocess.run(
+                docker_run_skopeo,
+                capture_output=True,
+                text=True,
+                check=True
+            )
+            return result.stdout.strip()
+        except subprocess.CalledProcessError as e:
+            raise Exception(f"Failed to run skopeo inspect using 'docker run' for image {image}. Error: {e.stderr.strip()}") from e
+        except FileNotFoundError:
+            raise Exception("docker is not installed on the system.")
+        except Exception as e:
```

Comment:
nit: if you are catching `Exception e` here and then immediately re-throwing it without adding more context, then you could just skip the entire `except Exception as e` clause.